spec/support/command_helpers.rb

```diff
@@ -34,6 +34,51 @@ module CommandHelpers # rubocop:disable Metrics/ModuleLength
     }
   }.freeze
 
+  def config_dir
+    "#{spec_directory}/dummy/.controlplane"
+  end
+
+  def config_file_prefix(extra_prefix = "")
+    prefix = "controlplane"
+    prefix += "-#{extra_prefix}" unless extra_prefix.nil? || extra_prefix.empty?
+
+    prefix
+  end
+
+  def configure_config_file(extra_prefix = "")
+    prefix = config_file_prefix(extra_prefix)
+
+    config_file = File.read("#{config_dir}/#{prefix}.yml")
+    config_file = config_file.gsub("{GLOBAL_IDENTIFIER}", dummy_test_app_global_identifier)
+
+    @@tmp_config_file = Tempfile.create(["#{prefix}-tmp-", ".yml"], config_dir) # rubocop:disable Style/ClassVars
+    @@tmp_config_file.write(config_file)
+    @@tmp_config_file.rewind
+
+    ENV["CONFIG_FILE_PATH"] = @@tmp_config_file.path
+  end
+
+  def delete_config_file
+    return unless @@tmp_config_file
+
+    File.delete(@@tmp_config_file.path)
+    @@tmp_config_file = nil # rubocop:disable Style/ClassVars
+  end
+
+  def temporarily_switch_config_file(extra_prefix = "")
```

Comment:
In some specs, we want to temporarily switch to a different config, but return to the main config after. That's what this method and the one below are for.